src/libostree/libostree-released.sym

```diff
@@ -542,6 +542,10 @@ LIBOSTREE_2018.9 {
   ostree_repo_get_default_repo_finders;
 } LIBOSTREE_2018.7;
 
+/* No new symbols in 2019.1 */
+
+/* No new symbols in 2019.2 */
```

Comment:
Just remembered from previous work, that `ostree_repo_get_bootloader` was added as a public API in `2019.2`. (I should have added it to `-devel` at the time, seeing the `-devel` file now). Diff where it was added: https://github.com/ostreedev/ostree/pull/1814/files#diff-23d4c8887c7dda91613c9451bf3fe7c0R6107

Reply:
Thanks! I updated it.